app/src/components/widgets/multiLevelWidgets/testCaseSearchTable/testExecutionsPromoBanner/testExecutionsPromoBanner.scss

```
@@ -0,0 +1,151 @@
+
+.external-link-icon {
+  display: inline-flex;
+  flex-shrink: 0;
+
+  svg {
+    width: 16px;
+    height: 16px;
+  }
+
+  svg path {
+    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)